webapp/src/Service/DOMJudgeService.php

```diff
@@ -832,4 +832,43 @@ public function getContestStats(Contest $contest): array
             ->getSingleScalarResult();
         return $stats;
     }
+
+    public function getTwigDataForProblemsAction(int $userid): array {
```

Comment:
`$userid` should be understood as `$teamid`, right?

Reply:
thanks, done